ui/app/AppLayouts/Wallet/popups/AddEditSavedAddressPopup.qml

```diff
@@ -45,7 +45,7 @@ StatusDialog {
         readonly property int validationMode: root.edit ?
                                          StatusInput.ValidationMode.Always
                                        : StatusInput.ValidationMode.OnlyWhenDirty
-        readonly property bool valid: addressInput.valid && nameInput.valid && root.address !== Constants.zeroAddress
+        readonly property bool valid: addressInput.valid && nameInput.valid
```

Comment:
As I remember, I added this check because you could have added the `zeroAddress`. However, that being a corner case, we better merge this fix without fixing entering the zero address yet, if that is a more complicated fix.